src/emulate.c

```diff
@@ -1313,6 +211,7 @@ static inline bool insn_is_unconditional_branch(uint8_t opcode)
     return false;
 }
 
+#if !RV32_HAS(JIT)
```

Comment:
Is it possible to unify hash function implementation? i.e., unify macro `HASH` and function `hash`.

Reply:
It is possible, but it may impact the performance of interpreter.

Reply:
Add FIXME/TODO comments within this pull request. Then, submit an issue on GitHub to address code consolidation.